.github/workflows/ci.yml

```diff
@@ -195,8 +195,11 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - uses: dtolnay/rust-toolchain@nightly
-      - run: cargo install cross
+
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
Small nit since this area is being touched: It's probably worth synchronizing with the [names for these](https://github.com/rustls/rustls/pull/1342/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R230-R233) in the `rustls` repository for consistency.

Reply:
For whatever reason the webpki CI arranges its YAML with different spacing than the Rustls' repo, and doesn't use `name:` attributes at all.

I'm going to leave this as-is to match that style but I agree it would make sense to synchronize these at some point.